src/search-backend/radio-search-backend.ts

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import type { SearchParams } from '../search-params';
+import type { Result } from '@internetarchive/result-type';
+import type { SearchServiceError } from '../search-service-error';
+import { SearchParamURLGenerator } from '../search-param-url-generator';
+import { BaseSearchBackend } from './base-search-backend';
+import type { SearchBackendOptionsInterface } from './search-backend-options';
+
+/**
+ * The RadioSearchBackend requests search results among radio captions from archive.org
+ */
+export class RadioSearchBackend extends BaseSearchBackend {
+  private servicePath: string;
+
+  constructor(options?: SearchBackendOptionsInterface) {
+    super(options);
+    this.servicePath =
+      options?.servicePath ?? '/services/search/beta/page_production';
+  }
+
+  /** @inheritdoc */
+  async performSearch(
+    params: SearchParams
+  ): Promise<Result<any, SearchServiceError>> {
+    if (this.debuggingEnabled && params.debugging === undefined) {
```

Comment:
can we do `!params.debugging` ?

Reply:
`debugging` is allowed to be explicitly set to `true` or `false` on a search, which overrides any global setting in the search service backend. So this line is just saying "if no `debugging` value was provided, use the default".

`!params.debugging` would change the logic, as it would no longer be possible to override the default by providing `false`.